### DISCUSSION TITLE
Typing `j` does not focus `jsonb` in column type selector

### DISCUSSION THREAD

**Discussion by probablykasper** [supabase/supabase#16940] [Contribute to Supabase]
# Bug report

Typing `j` does not focus anything in the type column type picker:

<img width="497" alt="image" src="https://github.com/supabase/supabase/assets/11315492/11ade2cf-0b36-401e-aac6-25cafdb758ee">

## System information

- OS: macOS
- Browser (if applies) Brave

**Comment by MildTomato** [comment]
This will be resolved in the shadcn component migration
